### ISSUE TITLE
Upgrade 6.1 Build Error - Unable to resolve module ./lib/plugin/PluginContainer

### ISSUE THREAD

**Issue by dl-husky73** [iamacup/react-native-markdown-display#69]
Just upgraded from 6.01 to 6.1.  Getting the following error:
error: Error: Unable to resolve module `./lib/plugin/PluginContainer` from `node_modules\react-native-markdown-display\src\index.js`:

Platform:
Android
RN: 62.0
RN-Markdown-display: 6.1

Specific Language is very straightforward...getting content from Strapi and diplaying markdown content.  Worked great prior to 6.1
import Markdown from 'react-native-markdown-display';
<Markdown>{item.content}</Markdown>

Steps taken:
Cleared Android Cache
Uninstall
Reinstall
Same error

**Comment by iamacup** [comment]
This really should not be happening because we removed that file in 6.1 - can you tell me what your yarn/package lock has in it relating to `react-native-markdown-display` ?

**Comment by dl-husky73** [comment]
I downgraded to 6.0.1?

 "react-native-markdown-display": {
      "version": "6.0.1",
      "resolved": "https://registry.npmjs.org/react-native-markdown-display/-/react-native-markdown-display-6.0.1.tgz",
      "integrity": "sha512-MZlW8BS/vjk9Ze4GTW60r/Begy+mrQMk7cy9XPTVKl8GxnAKp0Ps86D74/U1ED1YFLuHZg0cIeQh1/tlSWbaiw==",
      "requires": {
        "css-to-react-native": "^3.0.0",
        "markdown-it": "^10.0.0",
        "prop-types": "^15.7.2",
        "react-native-fit-image": "^1.5.5"
      }

**Comment by iamacup** [comment]
OK well the reason i ask is, it might not be resolving correctly or something - if you look at the source in https://registry.npmjs.org/react-native-markdown-display/-/react-native-markdown-display-6.1.0.tgz you will see that it does not reference `./lib/plugin/PluginContainer` in `index.js` so the error is probably something with your env / cache - although i can see you have cleared the android cache

To test this you need to update to 6.1.x and then go to your `node_modules\react-native-markdown-display\package.json` and see if it is actually 6.1, if it is at that point your metro bundler needs to be reset, [this might help](https://gist.github.com/jarretmoses/c2e4786fd342b3444f3bc6beff32098d)

**Comment by dl-husky73** [comment]
OK< I will try tonight

**Comment by dl-husky73** [comment]
Hi, I upgraded to RN 63 today and had this issue arise again.  I uninstalled RNMD, reinstalled, NPM Cache Clean --Force, Gradlew Clean and still have same issue.  
![error1](https://user-images.githubusercontent.com/47721681/87809904-27322200-c82a-11ea-9a08-032126b94127.JPG)


My code is as plain as can be:
import Markdown from 'react-native-markdown-display';
<Markdown>{article.content}</Markdown>

I have attached the error.  Seems pretty straight forward.  I don't know if others are having this issue.  I do know when I downgraded to a lower version it worked fine.  I went thru /src/index.js and no mention of plugin.  Maybe another dependency?  

I am not sure if I have some caching issue or did something weird, but could use your thoughts?
